models/product.js

```diff
@@ -4,6 +4,7 @@ function Product(options) {
   this.image = options['image'];
   this.description = options['description'];
   Product.all.push(this);
+  Product.push(cart);
 }
```

Comment:
Remember, you didn't need to add anything to this file per directions. Line 6 is taking care of adding the cart to the products array.